<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetCompra</class>
 <widget class="QWidget" name="WidgetCompra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Pendentes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="WidgetCompraPendentes" name="widgetPendentes" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Gerar Compra</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="WidgetCompraGerar" name="widgetGerar" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Confirmar Compra</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="WidgetCompraConfirmar" name="widgetConfirmar" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Faturamento</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="WidgetCompraFaturar" name="widgetFaturar" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetCompraGerar</class>
   <extends>QWidget</extends>
   <header>src/widgetcompragerar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCompraConfirmar</class>
   <extends>QWidget</extends>
   <header>src/widgetcompraconfirmar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCompraFaturar</class>
   <extends>QWidget</extends>
   <header>src/widgetcomprafaturar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCompraPendentes</class>
   <extends>QWidget</extends>
   <header>src/widgetcomprapendentes.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
